ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js

```diff
@@ -5,6 +5,7 @@ $(function() {
                 $('.signin').css('display','none');
                 $('.signup').css('display','none');
                 $('.logout').css('display','inline-block');
+				$('.myaccount').css('display','inline-block');
```

Comment:
@DeepaliAgrwl  My Account Button should be hidden for anonymous user.

Reply:
this is done